readme.md

```diff
@@ -45,6 +45,12 @@ The main purpose of this package is for **faster Test-driven Development**, it g
 
 ## How to install
 
+#### For Laravel 9.x
+
+```bash
+# Get the package
+$ composer require luthfi/simple-crud-generator:^3.0
```

Comment:
Hi @Leamsi9. We need to install this package as a dev dependency.

```suggestion
$ composer require luthfi/simple-crud-generator:^3.0 --dev
```
 Would you mind changing the other installation command as well (for 8.x)? Tanks